flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientTest.java

```diff
@@ -129,6 +142,126 @@ public void testHistoryFile() throws Exception {
         }
     }
 
+    @Test
+    public void testSqlFileContainsUnknownStatement() throws Exception {
```

Comment:
We have support the option `sql-client.verbose` now. Users can print the full stack now. 

Sql client just report the exception.. It's the Calcite parser that doesn't give us meaningful exception message. 

Maybe we can print `Could not parse SQL statement` if the parse fails. Otherwise print `Could not execute SQL statement`.